server/etcdserver/apply/apply.go

```diff
@@ -34,8 +35,6 @@ import (
 	serverstorage "go.etcd.io/etcd/server/v3/storage"
 	"go.etcd.io/etcd/server/v3/storage/backend"
 	"go.etcd.io/etcd/server/v3/storage/mvcc"
-
-	"github.com/gogo/protobuf/proto"
```

Comment:
minor comment, please organize the import entries into three groups something like as below,

```
import (
    "context"

    "go.etcd.io/X"
    "go.etcd.io/Y"

    "github.com/xxx"
    "go.uber.org/zap"
)
```